FaunaDB.Client/Query/Language.TimeDate.cs

```diff
@@ -51,5 +51,14 @@ public enum TimeUnit
         /// </summary>
         public static Expr Date(Expr date) =>
             UnescapedObject.With("date", date);
+
+        /// <summary>
+        /// Constructs a Timestamp representing the transaction’s start time.
+        /// <para>
+        /// See the <see href="https://docs.fauna.com/fauna/current/api/fql/functions/now">FaunaDB Now Functions</see>
+        /// </para>
+        /// </summary>
+        public static Expr Now() =>
```

Comment:
I think this doesn't belong to this PR :D

Reply:
properly rebased 🤦🏻‍♂️